store.go

```diff
@@ -2842,57 +2846,63 @@ func (s *store) mount(id string, options drivers.MountOpts) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	if options.UidMaps != nil || options.GidMaps != nil {
-		options.DisableShifting = !s.canUseShifting(options.UidMaps, options.GidMaps)
-	}
+	var imageHomeStore roImageStore // Set if image != ""
+	var cimage *Image               // Set if image != ""
```

Comment:
The `Set if image != ""` comments are not applicable here.

(Non-blocking nit: The variables could be moved immediately before their first use (or, in case of `cimage`, use :=`).)

Reply:
The primary thing I’d like to see fixed here is removing the incorrect/inapplicable comments.